### PR TITLE
Added conversion between translation and rotation.

### PR DESCRIPTION
**Pull request by niklasborwell**

### CODE REVIEW

File: source/math/FloatExtension.ooc [hunk 1]
```diff
@@ -120,4 +120,10 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	getTranslationFromRotation: static func (rotation: Float, zDistance: Float) -> Float {
+		sin(rotation) / zDistance
+	}
+	getRotationFromTranslation: static func (translation: Float, zDistance: Float) -> Float {
```
Comment (davidhesselbom) [inline source/math/FloatExtension.ooc]: `getRotationFromTranslation: static func (translation, zDistance: This) -> This {`

**Comment by niklasborwell** [comment]
Updated.

File: source/math/FloatExtension.ooc [hunk 2]
```diff
@@ -120,4 +120,10 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
+	getTranslationFromRotation: static func (rotation: This, zDistance: This) -> This {
```
Comment (davidhesselbom) [inline source/math/FloatExtension.ooc]: `getTranslationFromRotation: static func (rotation, zDistance: This) -> This {` didn't work?

Comment (niklasborwell) [inline source/math/FloatExtension.ooc]: oh shit. Missed that

**Comment by niklasborwell** [comment]
Updated again.

**Pull request merged**